src/components/Commons/InfoTooltip/InfoTooltip.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Tooltip from '@mui/material/Tooltip';
+import ClickAwayListener from '@mui/material/ClickAwayListener';
+import InfoRoundedIcon from '@mui/icons-material/InfoRounded';
+import { Typography } from '@mui/material';
+
+export const InfoTooltip = ({ title, texts, infoLink }) => {
+    const [open, setOpen] = useState(false);
+
+    const htmlTitle = () => {
+        return (
+            <React.Fragment>
+                <Typography color="inherit">{title}</Typography>
+                {texts.map(({ level, description }) => <p key={level}> <b>{level}:</b> {description}</p>)}
+                {infoLink && <b><a rel="noreferrer" target="_blank" style={{ color: 'white' }} href={infoLink}>more info...</a></b>}
+            </React.Fragment>
+        )
+    }
+
+    const handleTooltipClose = () => {
+        setOpen(false);
+    };
+
+    const handleTooltipOpen = () => {
+        setOpen(true);
+    };
+
+    return (
+        <div>
```

Comment:
Delete this div, replace it with a fragment

Reply:
Done. It contained the click away listener as the parent tag for all. So I just deleted the div.